core-web/libs/sdk/angular/README.md

```diff
@@ -1,229 +1,642 @@
+
+For a step-by-step guide on setting up the Universal Visual Editor, check out our [easy-to-follow instructions](https://dev.dotcms.com/docs/uve-headless-config) and get started in no time!
+
+### Create a dotCMS API Key
+
+> [!TIP]
+> Make sure your API Token has read-only permissions for Pages, Folders, Assets, and Content. Using a key with minimal permissions follows security best practices.
+
+This integration requires an API Key with read-only permissions for security best practices:
+
+1. Go to the **dotCMS admin panel**.
+2. Click on **System** > **Users**.
+3. Select the user you want to create the API Key for.
+4. Go to **API Access Key** and generate a new key.
+
+For detailed instructions, please refer to the [dotCMS API Documentation - Read-only token](https://dev.dotcms.com/docs/rest-api-authentication#ReadOnlyToken).
+
+### Installation
+
+```bash
+npm install @dotcms/angular@next @dotcms/uve@next @dotcms/client@next @dotcms/types@next @tinymce/tinymce-angular
 ```
 
-Add this configuration to `ApplicationConfig` in your Angular app.
+### dotCMS Client Configuration
 
-`src/app/app.config.ts`
 ```typescript
+import { createDotCMSClient } from '@dotcms/client';
 import { InjectionToken } from '@angular/core';
-import { ClientConfig, DotCmsClient } from '@dotcms/client';
 
-export const DOTCMS_CLIENT_TOKEN = new InjectionToken<DotCmsClient>('DOTCMS_CLIENT');
+export type DotCMSClient = ReturnType<typeof createDotCMSClient>;
+
+const dotCMSClient: DotCMSClient = createDotCMSClient({
+    dotcmsUrl: 'https://your-dotcms-instance.com',
+    authToken: 'your-auth-token', // Optional for public content
+    siteId: 'your-site-id' // Optional site identifier/name
+});
+
+export const DOTCMS_CLIENT_TOKEN = new InjectionToken<DotCMSClient>('DOTCMS_CLIENT');
 
 export const appConfig: ApplicationConfig = {
     providers: [
-        provideRouter(routes),
         {
             provide: DOTCMS_CLIENT_TOKEN,
-            useValue: DotCmsClient.init(DOTCMS_CLIENT_CONFIG),
+            useValue: dotCMSClient
         }
-    ],
+    ]
 };
 ```
 
-This way, we will have access to `DOTCMS_CLIENT_TOKEN` from anywhere in our application.
+This configuration makes the dotCMS client service available throughout your Angular application, allowing you to inject it wherever needed. For more details on how Angular's InjectionToken works, you can refer to the [Angular InjectionToken documentation](https://angular.dev/api/core/InjectionToken).
+
+### Proxy Configuration for Static Assets
+
+Configure a proxy to leverage the powerful dotCMS image API, allowing you to resize and serve optimized images efficiently. This enhances application performance and improves user experience, making it a strategic enhancement for your project.
 
-### Client Usage
-To interact with the client and obtain information from, for example, our pages
+#### 1. Create a Proxy Configuration
+
+Create a `proxy.conf.json` file in your project:
+
+```json
+// proxy.conf.json
```

Comment:
If we use use `proxy.conf.js` we can use the env variable for the host. For next version thing.